.github/workflows/react.yml

```diff
@@ -29,4 +29,5 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-node-
 
+      - run: CI=false make check-react-app
```

Comment:
Where is this `CI` environment variable used in code?

Reply:
It's used by npm to threat as errors, and set to true by default by github actions (I assume). Currently there are a few warnings in the react app that, so without this the build fails in CI. We previously didn't build the react-app in CI at all.

These should ideally be addressed, and I've added the following issue for fixing that meanwhile: https://github.com/thanos-io/thanos/issues/7066